app/controllers/attendances_controller.rb

```diff
@@ -1,4 +1,40 @@
 class AttendancesController < ApplicationController
   def index
   end
+
+  def new
+    @attendance = Attendance.new
+    @company = Company.where(disable: false)
+  end
+
+  def create
+    @employee = Employee.where(disable:false)
+    data = params.require(:attendance).permit(:private_num,:company_id)
+    time = Time.now.strftime("%H:%M:%S")
+    day = Time.now.strftime("%d-%m-%Y")
+    emp = @employee.find_by(private_num: data.fetch(:private_num), company_id: data.fetch(:company_id))
+
+    if !emp.nil?
```

Comment:
This whole `if` block could be replaced with a guard clause instead:

```ruby
return if emp.nil?

# stuff to do if emp is not nil goes here.
```

https://anthonygharvey.com/guard-clauses-vs-nested-conditionals